src/Notepads/Strings/cs-CZ/Manifest.resw

```diff
@@ -118,199 +118,200 @@
   </data>
   <data name="ProjectFileDisplayName" xml:space="preserve">
-    <value>Project File</value>
+    <value>Projectový soubor</value>
     <comment>.project, .prj</comment>
   </data>
   <data name="PythonFileDisplayName" xml:space="preserve">
-    <value>Python File</value>
+    <value>Soubor Python</value>
     <comment>.py</comment>
   </data>
   <data name="RubyFileDisplayName" xml:space="preserve">
-    <value>Ruby File</value>
+    <value>Soubor Ruby</value>
     <comment>.rb</comment>
   </data>
   <data name="RunCommandsFileDisplayName" xml:space="preserve">
-    <value>Run Commands File</value>
+    <value>Soubor spustitelných příkazů</value>
     <comment>.bashrc, .vimrc, .rc</comment>
   </data>
   <data name="ShellScriptFileDisplayName" xml:space="preserve">
-    <value>Shell Script File</value>
+    <value>Soubor skriptu Shell</value>
     <comment>.sh</comment>
   </data>
   <data name="SqlFileDisplayName" xml:space="preserve">
-    <value>SQL File</value>
+    <value>Soubor SQL</value>
     <comment>.sql</comment>
   </data>
   <data name="SubStationAlphaSubtitleFileDisplayName" xml:space="preserve">
-    <value>Sub Station Alpha Subtitle File</value>
+    <value>Soubor Sub Station Alpha Subtitle</value>
     <comment>.ssa</comment>
   </data>
   <data name="SubtitleFileDisplayName" xml:space="preserve">
-    <value>Subtitle File</value>
+    <value>Soubor titulků</value>
     <comment>.srt</comment>
   </data>
   <data name="TextFileDisplayName" xml:space="preserve">
-    <value>Text Document</value>
+    <value>Textový soubor</value>
     <comment>.txt</comment>
   </data>
   <data name="TypeScriptFileDisplayName" xml:space="preserve">
-    <value>TypeScript File</value>
+    <value>Soubor TypeScript</value>
     <comment>.ts</comment>
   </data>
   <data name="VisualBasicFileDisplayName" xml:space="preserve">
-    <value>Visual Basic File</value>
+    <value>Soubor Visual Basic</value>
     <comment>.vb</comment>
   </data>
   <data name="VueConfigFileDisplayName" xml:space="preserve">
-    <value>Vue Config File</value>
+    <value>Konfigurační soubor Vue</value>
     <comment>.vuerc</comment>
   </data>
   <data name="VueFileDisplayName" xml:space="preserve">
-    <value>Vue File</value>
+    <value>Soubor Vue</value>
     <comment>.vue</comment>
   </data>
   <data name="XmlFileDisplayName" xml:space="preserve">
-    <value>XML File</value>
+    <value>Soubor XML</value>
     <comment>.xml</comment>
   </data>
   <data name="YamlFileDisplayName" xml:space="preserve">
-    <value>YAML File</value>
+    <value>Soubor YAML</value>
     <comment>.yml, .yaml</comment>
   </data>
   <data name="ZshellConfigFileDisplayName" xml:space="preserve">
-    <value>Z shell Config File</value>
+    <value>Konfigurační soubor Z shell</value>
     <comment>zshrc</comment>
   </data>
   <data name="ZshellHistoryFileDisplayName" xml:space="preserve">
-    <value>Z shell History File</value>
+    <value>Soubor historie Z shell</value>
+
```

Comment:
Can you help remove this empty space?

Reply:
Done

Reply:
@JanRajnoha  I just added few more string tonight. Can you sync your branch with master to make below changes?:
![image](https://user-images.githubusercontent.com/1166162/81400694-9381fe00-90e2-11ea-81ec-61a5ecbfa04c.png)

Reply:
Done